StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -127,5 +148,28 @@ private static void AppendFakeClintBoulderSpecialOrderKey()
             var specialOrderKeys = specialOrderKeysField.GetValue();
             specialOrderKeys.Add("Clint2Again");
         }
+
+        public static void ChangeScheduleForMovie()
+        {
+            if (!_archipelago.SlotData.Mods.HasMod(ModNames.SVE))
+            {
+                return;
+            }
+            if (_archipelago.GetReceivedItemCount("Progressive Movie Theater") >= 2)
+            {
+                string[] charactersToModify = {"Claire", "Martin"};
+                foreach(var name in charactersToModify)
+                {
+                    var schedules = Game1.content.Load<Dictionary<string, string>>($"Characters\\Schedules\\{name}");
+                    foreach (KeyValuePair<string, string> day in schedules)
+                    {
+                        if (characterToSchedule[name].Keys.Contains(day.Key))
+                        {
+                            schedules[day.Key] = characterToSchedule[name][day.Key];
```

Comment:
This is vulnerable to race conditions. You should not be modifying a dictionary while iterating on it